core/trino-main/src/main/java/io/trino/operator/OutputSpoolingOperatorFactory.java

```diff
@@ -343,7 +359,7 @@ static long reduce(List<Page> page, ToLongFunction<Page> reduce)
         public void close()
                 throws Exception
         {
-            userMemoryContext.close();
+            aggregatedMemoryContext.close();
```

Comment:
I think you do not need to do that

Reply:
I'm not sure about that tbh. Is it closed automatically by the driver?